app/post.go

```diff
@@ -577,6 +577,58 @@ func (a *App) GetPostsAroundPost(postId, channelId string, offset, limit int, be
 	}
 }
 
+func (a *App) GetPostAfter(channelId string, time int64) (*model.Post, *model.AppError) {
+	result := <-a.Srv.Store.Post().GetPostAfter(channelId, time)
+	if result.Err != nil {
+		return nil, result.Err
+	}
+
+	return result.Data.(*model.Post), nil
+}
+
+func (a *App) GetPostBefore(channelId string, time int64) (*model.Post, *model.AppError) {
+	result := <-a.Srv.Store.Post().GetPostBefore(channelId, time)
+	if result.Err != nil {
+		return nil, result.Err
+	}
+
+	return result.Data.(*model.Post), nil
+}
+
+func (a *App) GetNextPostFromPostList(postList *model.PostList) string {
```

Comment:
Can you rename this to `GetNextPostFromPostList` to make it clear that this returns an id and not the whole post?